spec/syntax.md

```diff
@@ -91,159 +91,198 @@ The syntax specification takes into account the following design restrictions:
 
 _This section is non-normative._
 
-### Messages
-
-All messages, including simple ones, are enclosed in `{…}` delimiters:
+### Message
 
-    {Hello, world!}
+A **_message_** is the complete template for a specific message formatting request.
 
-The same message defined in a `.properties` file:
+All _messages_, including simple ones, begin with U+007B LEFT CURLY BRACKET `{` 
```

Comment:
Is this accurate though?

A message can contain declarations, that start with `let $foo = {...}`,
or can contain a with a selector, starting with `match`.
In these cases the message does not start with `{`

The text below in the document seems to also consider these elements (declarations & selector) as part of the message:

> A _message_ containing a _declaration_ defining a local variable `$whom` which is then used twice inside the pattern:
>
> `let $whom = {$monster :noun case=accusative}`
> ...

and

> A message can have multiple selectors.
> A message with a single _selector_:
> 
> `match {$count :number}`

Reply:
You are correct: we start in code mode. Note that this is not text that I originated in this PR, so we've had a spec bug for a while.

Reply:
See if 7bd78fa fixes it.